### PR TITLE
set up donation page

### PR DESCRIPTION
**Pull request by pranjali-sharma**

### CODE REVIEW

File: src/main/webapp/donations.html [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+<head>
+    <script src="http://code.jquery.com/jquery-3.3.1.min.js"></script> <script src="https://code.jquery.com/ui/1.12.0/jquery-ui.min.js"></script>  <link rel="stylesheet" href="http://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css"> 
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
+
+<script>
+    //list of orgs+deets
+
+</script>
+
+<style>
```
Comment (sbahaud) [inline src/main/webapp/donations.html]: Move style to the css file

File: src/main/webapp/donations.html [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
+.title{
```
Comment (sbahaud) [inline src/main/webapp/donations.html]: Add single space before {

File: src/main/webapp/donations.html [hunk 3]
```diff
@@ -1,3 +1,52 @@
...
+                <div class="w-100"></div>
+
+                <div class="col"><a href="https://secure.actblue.com/donate/ab_mn">Act Blue Racism and Police Brutality Funds</a></div>                
+                <div class="col">Single donations will be split between multiple organizations, with the ability to adjust what goes where.</div>
+                <div class="w-100"></div>
+
+                <div class="col"><a href="https://wearemrrc.com/">Minnesota Rapid Response Coalition; Twin Cities, Minnesota</a></div>                
+                <div class="col">Donations will go toward rebuilding businesses and other parts of Black communities where protests have occurred and/or have been hit particularly hard by the coronavirus pandemic.</div>
+                <div class="w-100"></div>
+                </div>
```
Comment (sbahaud) [inline src/main/webapp/donations.html]: nit: Fix indent for \</div>

**Review by sbahaud — APPROVED** [review]
Approved with minor comments

**Review by kierabr1 — APPROVED** [review]

**Pull request merged**